modules/core/src/main/scala/org/tessellation/Main.scala

```diff
@@ -24,78 +22,90 @@ import org.tessellation.schema.node.NodeState
 import org.tessellation.schema.peer.PeerId
 import org.tessellation.security.SecurityProvider
 
+import com.monovore.decline.Opts
+import com.monovore.decline.effect.CommandIOApp
 import org.typelevel.log4cats.slf4j.Slf4jLogger
 
-object Main extends IOApp {
+object Main
+    extends CommandIOApp(
+      name = "",
+      header = "Constellation Node",
+      version = "0.0.x"
```

Comment:
It would be a cumbersome to keep version both in `build.sbt` as well as in `Main.scala`. I think we could use [sbt-buildinfo](https://github.com/sbt/sbt-buildinfo) plugin to keep them in sync. Could you create a separate task for using this plugin?

Reply:
I already did that yesterday and posted on slack @tbekas :)

Reply:
Awesome :)